src/desktop-lite/install.sh

```diff
@@ -411,7 +411,7 @@ else
 fi
 
 # Run whatever was passed in
-log "Executing "\$@\"."
+log "Executing \"$@\"."
```

Comment:
Can you bump the patch version of this Feature? Thanks! 

https://github.com/devcontainers/features/blob/01c0c4b76721aaedfaf450a1d6c0b497eaf6d916/src/desktop-lite/devcontainer-feature.json#L3

Reply:
Let me know when it's ready for another review